### DISCUSSION TITLE
How can we ignore a PVC from being resized?

### DISCUSSION THREAD

**Discussion by umeshbitgo** [topolvm/topolvm#967] [Q&A]
I wanted a few PVCs to be ignored by auto-resizer. is there an annotation to ignore PVC? I initially thought if we don't annotate the PVC it will be ignored by auto-resizer but this part in the doc got me confused: 
The PVC can optionally have resize.topolvm.io/threshold, resize.topolvm.io/inodes-threshold and resize.topolvm.io/increase annotations. (If they are not given, the default value is 10%.). does it mean that all PVCs in the clusters will be considered for resizing even if they aren't annotated because auto-resizer will use the default value as 10%?

**Comment by toshipp** [comment]
I guess you intended to create this discussion in pvc-autoresizer. Could you re-create the discussion?
https://github.com/topolvm/pvc-autoresizer/discussions
